src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -792,6 +882,25 @@ class DevTask extends AbstractServerTask {
         }
     }
 
+    private void processContainerParams() throws Exception {
```

Comment:
If setDockerfile and setDockerRunOpts called setContainer(true), this would not be necessary right? Should still check that dockerfile.exists() though.

Reply:
Refactored to put processContainerParams into initializeDefaultValues, and moved the other container related param processing into there as well. Commit: https://github.com/OpenLiberty/ci.gradle/pull/477/commits/e56e331c5e9569bae1cc2daeac79daba72cc5b08